tools/test-proxy/Azure.Sdk.Tools.TestProxy.Tests/TestHelpers.cs

```diff
@@ -39,6 +39,14 @@ public static RecordingHandler LoadRecordSessionIntoInMemoryStore(string path)
             return handler;
         }
 
+        public static string GenerateStringFromStream(Stream s)
+        {
+            s.Position = 0;
+            StreamReader reader = new StreamReader(s);
```

Comment:
```suggestion
            using StreamReader reader = new StreamReader(s);
```

`StreamReader` implements `IDisposable` so it should be disposed when no longer needed.